pkg/ship/kustomize.go

```diff
@@ -63,15 +65,29 @@ func (s *Ship) Init(ctx context.Context) error {
 				return warnings.WarnCannotRemoveState
 			}
 
-			if err := s.promptToRemoveState(); err != nil {
-				debug.Log("event", "state.remove.prompt.fail")
-				return err
+			if removeExistingState {
+				if err := s.promptToRemoveState(); err != nil {
+					debug.Log("event", "state.remove.prompt.fail")
+					return err
+				}
+			} else {
+				s.UI.Info(fmt.Sprint("Preserving current state"))
```

Comment:
Is there any reason to wrap this in `fmt.Sprint`?

Reply:
no, probably an artifact of old changes